### PR TITLE
Desktop Wallet release Sept 2023

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Inform readers about the contents of the desktop wallet release.

## Changes

Release notes, downloads, variables

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

**Comment by dg-concordium** [comment]
@orhoj and @shjortConcordium all I need is the final packages uploaded so I can update checksums.

File: source/mainnet/net/desktop-wallet/install-ledger-app.rst
Comment (shjortConcordium) [inline source/mainnet/net/desktop-wallet/install-ledger-app.rst:1]: There are some places in the macOS section that we still refer to `concordium-ledger-app-2.0.1`, which probably could be changed.

**Review by shjortConcordium — APPROVED** [review]
LGTM

File: source/mainnet/variables.rst [hunk 1]
```diff
@@ -1,9 +1,9 @@
 .. Desktop wallet verification variables
-.. |cdw-appimage| replace:: concordium-desktop-wallet-1.5.0.AppImage
+.. |cdw-appimage| replace:: concordium-desktop-wallet-1.6.0.AppImage
 .. |cdw-appimage-checksum| replace:: a04dcab599799533d8e35765a656a0ad938074d5f267f1829c5aa8258376b4e8
```
Comment (shjortConcordium) [inline source/mainnet/variables.rst:3]:
```suggestion
.. |cdw-appimage-checksum| replace:: ebc8082fa938ebce78fc407eaa0b9f79bc9da71d6d36ecde6c769decdf4f52b6
```

File: source/mainnet/variables.rst [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-.. |cdw-deb| replace:: concordium-desktop-wallet-1.5.0.deb
+.. |cdw-deb| replace:: concordium-desktop-wallet-1.6.0.deb
 .. |cdw-deb-checksum| replace:: ef0237f30e1c435323ef2eff2df82a23ffbb54b7c0b329666a07c78fb61119f1
```
Comment (shjortConcordium) [inline source/mainnet/variables.rst:5]:
```suggestion
.. |cdw-deb-checksum| replace:: ca9f4e38a422ba64e07a89e38cc762005bc31f3f4cbb399657a0adbd0de25a0e
```

File: source/mainnet/variables.rst [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-.. |cdw-rpm| replace:: concordium-desktop-wallet-1.5.0.rpm
+.. |cdw-rpm| replace:: concordium-desktop-wallet-1.6.0.rpm
 .. |cdw-rpm-checksum| replace:: 845f375c728b14ce13400869001b58c894510454dd7af8fa95d9538b5d8ef2d4
```
Comment (shjortConcordium) [inline source/mainnet/variables.rst:7]:
```suggestion
.. |cdw-rpm-checksum| replace:: 92108b193bc314ba5dc3bd7fe25626645b70d1f7f5f8c0c85fd30aeccc67350f
```

**Pull request merged**